### ISSUE TITLE
H1 not showing focus indicator on page load

### ISSUE THREAD

**Issue by outerpress** [department-of-veterans-affairs/va.gov-team#103524]
## Summary
The H1 on each page should receive focus and display a visible focus indicator when the page loads. The H1s are receiving focus, but not showing the visible focus indicator.

It looks like the focus indicator is being removed in vaos.css

<img width="682" alt="Image" src="https://github.com/user-attachments/assets/3f4af8cc-5de6-40d8-a368-33166da3d3dd" />

[Slack thread where this came up](https://dsva.slack.com/archives/C03CGTDLTFF/p1739895449709279). FYI @rileyorr 

## Specs
- Device: Mac Powerbook Pro
- Browser: Chrome

## Steps to Reproduce
**Test User:** 

1. Log in as Judy
2. Navigate to the Appointments tool
3. Note that the H1 "Appointments" does not have a visible focus indicator
4. Navigate to Messages
5. Note that the H1 "Messages" does have a visible focus indicator. This is the correct state

### Actual Result

H1 has focus, but no visible focus indicator:

<img width="414" alt="Image" src="https://github.com/user-attachments/assets/81fc74eb-b764-4ce2-80f1-40f5b7b14c4b" />

### Desired Result

H1 should have focus, and have a visible focus indicator, similar to the Messages tool:

<img width="414" alt="Image" src="https://github.com/user-attachments/assets/84ede85f-4e70-4a63-a304-01fd753be572" />

## Definition of Done
- [ ] Desired result is achieved and/or document decisions made
- [ ] Relevant documentation is updated

---
## How to configure this issue
- [ ] **Labeled with Team** (`appointments`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `design`, `product`)
- [ ] **Labeled with** `bug`
- [ ] **Assigned to** Product for prioritization

**Comment by rileyorr** [comment]
@outerpress I tested this in staging on both desktop and mobile and it looks great!
